src/System.Memory/src/System/Buffers/SequenceReader_search.cs

```diff
@@ -425,13 +427,13 @@ private bool TryReadToAnyInternal(out ReadOnlySequence<T> sequence, ReadOnlySpan
         }
 
         /// <summary>
-        /// Try to read data until the given <paramref name="delimiter"/> sequence.
+        /// Try to read data until the given <paramref name="delimiter"/>.
```

Comment:
Should we explicitly mention here that we are reading until the entire delimiter rather than any of the delimiters (maybe as a remark)?

Reply:
Hmm. To met it is pretty clear given the Any apis, but I'll update it.